hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/grantapproval/AbstractGrantApprovalCall.java

```diff
@@ -70,16 +67,8 @@ public TransactionBody callGrantApproval() {
             var ownerId = getOwnerId();
 
             if (ownerId != null && !isNftApprovalRevocation()) {
-                List<AccountApprovalForAllAllowance> accountApprovalForAllAllowances = enhancement
-                        .nativeOperations()
-                        .getAccount(ownerId.accountNum())
-                        .approveForAllNftAllowances();
-                if (accountApprovalForAllAllowances != null) {
-                    for (var approvedForAll : accountApprovalForAllAllowances) {
-                        if (approvedForAll.tokenId().equals(token)) {
-                            return buildCryptoApproveAllowance(approveDelegate(ownerId, approvedForAll.spenderId()));
-                        }
-                    }
```

Comment:
Why were these lines removed?

Reply:
This checks will be done in `dispatch()` to `CryptoApproveAllowanceHandler`.